bin/daemon.php

```diff
@@ -71,15 +71,14 @@
 
 DI::mode()->setExecutor(Mode::DAEMON);
 
-DI::config()->load();
+DI::config()->reload();
 
 if (empty(DI::config()->get('system', 'pidfile'))) {
 	die(<<<TXT
-Please set system.pidfile in config/local.config.php. For example:
+Please set system.pidfile in config/local.config.neon. For example:
     
-    'system' => [ 
-        'pidfile' => '/path/to/daemon.pid',
-    ],
+    'system:
+        pidfile: /path/to/daemon.pid
```

Comment:
The removal of the `return []` structure of the `config/local.config.php` file means that categories and keys are defined at one indentation level fewer than before.
```suggestion
system:
	pidfile: /path/to/daemon.pid
```